plugins-ui/proto/policy.proto

```
@@ -0,0 +1,109 @@
+syntax = "proto3";
+
+package types;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "rule.proto";
```

Buf (1.55.1)

import "rule.proto": file does not exist

(COMPILE)